hawkbit-dmf/hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/integration/AbstractAmqpServiceIntegrationTest.java

```diff
@@ -235,7 +235,7 @@ protected Long cancelAction(final Long actionId, final String controllerId) {
 
     protected Long registerTargetAndCancelActionId(final String controllerId) {
         final DistributionSetAssignmentResult assignmentResult = registerTargetAndAssignDistributionSet(controllerId);
-        return cancelAction(assignmentResult.getActionIds().get(0), controllerId);
+        return cancelAction(assignmentResult.getAssignedActions().get(0).getId(), controllerId);
```

Comment:
check if `assignedActions()` are used as objects somewhere, otherwise, store only the ids